### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#144] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-09 172713](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/590b7c45-dcc3-4ac2-a571-1b1fabf4352d)

**Comment by daromacarlo** [comment]
![Immagine 2023-12-17 152428](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/1d9b3798-6e10-47f3-a178-8fd170e4ec4f)
![Immagine 2023-12-17 152444](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/9de256cc-7a50-4a2a-aae7-ba49ee60c03a)
![Immagine 2023-12-17 152500](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/6bc3c040-eaa8-4d05-a1fd-e65f62807ec5)
![Immagine 2023-12-17 152516](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/68d419f7-56b8-4399-868b-9983274b86e5)

**Comment by daromacarlo** [comment]
punto 3 sbagliato.

**Reply by daromacarlo** [reply]
![Immagine 2024-01-18 114420](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/8496ed21-d2a8-45c7-81de-63d7f1366198)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/2f6c12b9-2306-4814-9991-92df5237ab4c)
